keyboards/bbrfkr/dynamis/dynamis.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2022 bbrfkr
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "dynamis.h"
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
```

Comment:
```suggestion
#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) {
        return false; 
    }
```

Reply:
I changed it, and add `#endif`.

Reply:
This still needs to be applied.  Specifically, the function needs to be the kb version of it, with a check for the user level function.  Otherwise, if/when a user wants to customize it... it will error out when compiling.

Reply:
ok. I'll apply this suggestion.